### DISCUSSION TITLE
AOC Day 19 - Best part of The Core

### DISCUSSION THREAD

**Discussion by rhdeck** [Virtual-Coffee/virtualcoffee.io#787] [Advent of Code 2022]
Discussion of solutions for [Advent of Code Day 19](https://adventofcode.com/2022/day/19) here.

Don't forget about the VC Advent of Code private leaderboard:

Go to [leaderboard/private](https://adventofcode.com/2022/leaderboard/private)
Use join code 363496-e6f42283

**Comment by rhdeck** [comment]
I love that a heuristic approach could save me again. Not computationally ideal, but [it got the job done.](https://github.com/rhdeck/aoc-2022/blob/main/src/19/day_19.ipynb)

**Comment by doleraj** [comment]
Just empty-brain BFS here, with a culling to the 1 million top states between minutes to save my heap space. But I learned about generators, I guess?

https://github.com/doleraj/adventofcode/blob/main/aoc2022/src/day19/index.ts
